### DISCUSSION TITLE
Mejorar búsqueda de ubicaciones

### DISCUSSION THREAD

**Discussion by EdgarOns** [breezy-weather/breezy-weather#1328] [Ideas]
### Describe your idea

1. Searching for locations using databases is always limited and does not allow obtaining weather forecasts for all corners of the planet. For this reason, I suggest that just as other weather applications have done (Today Weather, WeaWow, Rain Weather, etc.), to give it a more modern touch, the ideal thing would be to allow adding locations directly through a map, like Google or similar, in a way that we will be able to have meteorological data for any place on earth. 2. It is necessary to update the Open-Meteo API data to its latest version. I say this since parameters such as barometric pressure are not accurate in mountain areas or above a certain altitude. However, Open-Meteo offers this reliable data in other applications that use its source code and API. 3. Include new weather icon packs, to satisfy different user tastes. 4. Add more weather parameters in the application on the main screen, such as wind gusts, visibility, solar radiation, local time of the location, percentage of precipitation, distinction between rain or snow, etc. Open-Meteo offers all of them.

1. La búsqueda de ubicaciones mediante bases de datos siempre es limitada y no permite obtener pronósticos meteorológicos para todos los rincones del planeta. Por tal motivo, sugiero que así como han hecho otras aplicaciones meteorológicas (Today Weather, WeaWow, Rain Weather, etc) para darle un toque más moderno lo idóneo sería permitir añadir ubicaciones directamente a través de un mapa, tipo google o similar, de forma que podremos disponer de datos meteorológicos para cualquier lugar en la tierra.

2. Es necesario actualizar los datos de la API de Open-Meteo a su última versión. Esto lo digo ya que parámetros como la presión barométrica no es precisa en zonas de montaña o a partir de cierta altitud. Sin embargo, Open-Meteo ofrece esos datos fiables en otras aplicaciones que jsan su código fuente y API.

3. Incluir nuevos paquetes de iconos meteorológicos, para satisfacer los diferentes gustos de los usuarios.

4. Añadir más parámetros meteorológicos en la aplicación en la pantalla principal, como por ejemplo rachas de viento, visibilidad, radiación solar, hora local de la ubicación, porcentaje de precipitación, distinción entre lluvia o nieve, etc. Open-Meteo ofrece todos ellos.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
One idea per issue, please.
1) is duplicate of #10, no need to reopen it
